srv/salt/ceph/stage/radosgw/default.sls

```diff
@@ -19,3 +21,4 @@ rgw users:
     - sls: ceph.rgw
 
 {% endfor %}
+{% endif %}
```

Comment:
Stylistically, I think it best if the spacing around the jinja in all files is consistent. I might suggest we standardize on one newline before and one after jinja lines.